src/configuration/configuration.ts

```diff
@@ -8,10 +8,14 @@ export class Configuration {
         return this.get('codemaker.apiKey');
     }
 
-    static isAutoCompleteEnabled(): boolean {
+    static isAutocompleteEnabled(): boolean {
         return this.get('codemaker.enableAutocomplete');
     }
 
+    static isAllowMultiLineAutocomplete(): boolean {
```

Comment:
Side note, for single line completion, we can add `\n` as one of the stop sequence in the generator